monitoring/grafana/dashboards/pool-detail.json

```diff
@@ -257,7 +257,7 @@
       "thresholds": [],
       "timeFrom": null,
       "timeShift": null,
-      "title": "$pool_name Object In-/Egress",
+      "title": "$pool_name Object In-Progress",
```

Comment:
I think this means `Ingress` (input data traffic) + `Egress` (output data traffic).

Reply:
Yes. I have changed it to "Object Ingress/Egress"